cmd/octillery/main.go

```diff
@@ -377,13 +377,14 @@ func (cmd *ImportCommand) Execute(args []string) error {
 		}
 		seeds, err := os.Open(path)
 		if err != nil {
-			return errors.WithStack(err)
+			return errors.Wrapf(err, "fail to open file %s", path)
```

Comment:
Could you replace error message to '**failed** to open file %s' ?

Reply:
Yes, now I fix it.